shared/signup/email/index.tsx

```diff
@@ -20,22 +19,21 @@ const EnterEmail = (props: Props) => {
 
   return (
     <SignupScreen
-      banners={errorBanner(props.error ? props.error.message : '')}
+      banners={errorBanner(props.error)}
       buttons={[
         {
-          disabled,
+          disabled: disabled || props.waiting,
```

Comment:
Hm, but I think this one should get `waiting`. You're waiting for this button's action to finish. Whereas we want to disable canceling while this is in flight.